### ISSUE TITLE
Downgraded JuMP to 0.21.2 as specified by this library and getting LoadError: LoadError: UndefVarError: JuMPTypes not defined

### ISSUE THREAD

**Issue by DrVogt** [anriseth/MultiJuMP.jl#44]
I downgraded jump to 0.21.2 and get

LoadError: LoadError: UndefVarError: JuMPTypes not defined

I even tried a clean reinstall of jump to 0.21.2 and still get this issue. When I search online I see you guys made the library work with 0.21.2 to fix this exact same error message, but it seems to maintain for me. Any tips guys?

**Comment by ArnaudKul** [comment]
Dear DrVogt,
I had the same problem.  It seems that it is due to the fact that the 'Project.toml' file indicates ' version = "0.5.0" '.  I solved the issue by copying the package in a personal directory (ArnaudKul/MultiJuMP.jl), where I could changed the file from version = "0.5.0" to version = "0.6.0".  And obviously, I added the package through the git adress of my own repository.  It works for me now.
Arnaud
